great_expectations/expectations/metrics/util.py

```diff
@@ -7,8 +7,10 @@
 from packaging import version
 
 from great_expectations.execution_engine.util import check_sql_engine_dialect
+from great_expectations.execution_engine.sqlalchemy_dialect import GESqlDialect
 from great_expectations.util import get_sqlalchemy_inspector
```

Comment:
This should make isort happy:
```suggestion
from great_expectations.execution_engine.sqlalchemy_dialect import GESqlDialect
from great_expectations.execution_engine.util import check_sql_engine_dialect
from great_expectations.util import get_sqlalchemy_inspector
```